compiler/src/iree/compiler/Codegen/LLVMGPU/test/gpu_set_num_workgroups.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: iree-opt --split-input-file --pass-pipeline="builtin.module(hal.executable(hal.executable.variant(iree-codegen-materialize-user-configs, iree-llvmgpu-select-lowering-strategy)))" \
+// RUN: iree-opt --split-input-file --pass-pipeline="builtin.module(hal.executable(hal.executable.variant(iree-codegen-llvmgpu-configuration-pipeline, iree-llvmgpu-select-lowering-strategy)))" \
```

Comment:
`iree-codegen-llvmgpu-configuration-pipeline` should run `iree-llvmgpu-select-lowering-strategy` as the last pass, so you can just drop the selection pass (same for all the other instances).

Reply:
Ah, good point. I was not aware of it, thanks!